specs/adrs/051024-Database-Choice.md

```diff
@@ -4,7 +4,7 @@ parent: Decisions
 nav_order: 1
 title: ADR 0001 - Choosing the Database Technology
 status: proposed
-date: 2024-05-10
+date: 2023-05-10
```

Comment:
why is this 2023? shouldn't it be 2024?

Reply:
yeah it should be, all of them were 2023 before. i think this was just an error, ill fix it.